pkg/operator/operator.go

```diff
@@ -21,6 +22,7 @@ import (
 	operatorv1 "github.com/openshift/api/operator/v1"
 
 	"k8s.io/client-go/rest"
+	"github.com/openshift/library-go/pkg/operator/events"
```

Comment:
order

Reply:
Addressed in the latest version of the PR.